### DISCUSSION TITLE
Markdown blue note box

### DISCUSSION THREAD

**Discussion by KoB-Kirito** [community/community#66895] [Other Feature Feedback, Questions, & Ideas]
How do I get that blue note box you use everywhere?
![image](https://github.com/community/community/assets/51691757/ed14dc04-03c7-4cd9-9e81-616467004e4b)

**Comment by seyLu** [comment]
Hi @KoB-Kirito,

I don't think it is possible to do that using Github Flavored Markdown. The closest thing is:

> [!NOTE]
> Highlights information that users should take into account, even when skimming.

<br>

But it is possible, in other markdown flavors. For example, if you are building a documentation site using [mkdocs-material](https://squidfunk.github.io/mkdocs-material/), you can do something similar like that in markdown by:

```md
!!! note

    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla et euismod
    nulla. Curabitur feugiat, tortor non consequat finibus, justo purus auctor
    massa, nec semper lorem quam in massa.
```

which then renders to:
![image](https://github.com/community/community/assets/98249191/b26d274b-7445-479e-a89c-f78a361b6ede)

Hope that helps.

Btw, nce pfp. Was the username, kirito, already taken?

**Reply by KoB-Kirito** [reply]
Of course it was taken xD

**Reply by KoB-Kirito** [reply]
I want to edit my readme.md, I guess I can't use that there?

**Reply by seyLu** [reply]
> Of course it was taken xD

Ahhh bummer. But I think you could've gotten it. This [kirito](https://github.com/kirito) here created the account at 2020.

> I want to edit my readme.md, I guess I can't use that there?

Sadly no, as also confirmend by @spenserblack. What you can do though is use a docs builder. And in your readme point the users for instructions to your docs. Your readme might contain some information, steps like setup instructions but usage is on the docs.

Example repos that does this:
- Ruff [[Github]](https://github.com/astral-sh/ruff) [[Docs]](https://beta.ruff.rs/docs/)
- Black [[Github]](https://github.com/psf/black) [[Docs]](https://black.readthedocs.io/en/stable/)
- Pytest [[Github]](https://github.com/pytest-dev/pytest) [[Docs]](https://docs.pytest.org/en/stable/)

Example Docs builder you might want to use:
- [mkdocs-material](https://squidfunk.github.io/mkdocs-material/)
- [vitepress](https://vitepress.dev/)
- [docosaurus](https://docusaurus.io/)
- [docsify](https://docsify.js.org/#/quickstart)

**Comment by spenserblack** [comment]
To add to @seyLu's answer, the GitHub docs use their own Markdown renderer for their docs website. [They use `{% note %}foo{% endnote %}` with this custom rendering](https://github.com/github/docs/blob/5c89fbca8b12ed2f9d3d860b7a31c7da8d11df5b/content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md?plain=1#L110C4-L117).

The tag is transformed [here](https://github.com/github/docs/blob/5c89fbca8b12ed2f9d3d860b7a31c7da8d11df5b/src/content-render/liquid/spotlight.js#L1-L14).

That's probably TMI, but basically:

- If you want to do that in Markdown rendered in a GitHub issue, `README.md`, etc. you can't (or you can hack it with an image :wink:)
- You can get something similar with
  ```markdown
  > [!NOTE]
  > This is a note
  ```
  > [!NOTE]
  > This is a note

  But this is in beta and the syntax might change (it used to be `> **Note**`)
- If you're looking to do that for a website or something, then search Markdown renderers that support admonitions.

**Reply by KoB-Kirito** [reply]
Can I do that on the project readme.md?

**Reply by spenserblack** [reply]
> [!NOTE]
> If you're talking about this style, yes, you can add that to a readme.md. [Example](https://github.com/spenserblack/test-repo/blob/18912c4a025c103ae75dc53bbfe03609df5c9100/README.md)

**Reply by KoB-Kirito** [reply]
No, the custom rendering.
I know the [!NOTE] style and don't like it.

**Reply by spenserblack** [reply]
:laughing: Me neither

But I'm afraid not. You'd want to think about deploying your docs to a website, like github.io, for more advanced styling.

There's some styling available by putting HTML in your Markdown files, like this:
```markdown
<div align="center">Note</div>
```
<div align="center">Note</div>

But the available HTML syntax is limited, and a lot of it will get cleaned up. But centering is one way to draw a bit more attention to some text.
